tgui/packages/tgui/interfaces/BotFloor.js

```diff
@@ -0,0 +1,103 @@
+import { useBackend } from '../backend';
+import {
+  Button,
+  LabeledList,
+  NoticeBox,
+  Section,
+  Box,
+} from '../components';
+import { ButtonCheckbox } from '../components/Button';
+import { Window } from '../layouts';
+import { BotStatus } from './common/BotStatus';
+
+export const BotFloor = (props, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    noaccess,
+    painame,
+    hullplating,
+    replace,
+    eat,
+    make,
+    fixfloor,
+    alert,
+    magnet,
+    tiles,
+  } = data;
+  return (
+    <Window resizable>
+      <Window.Content scrollable>
+          <BotStatus/>
```

Comment:
is this meant to be indented like this?

Reply:
ye, I made "<BotStatus/>" indent and then the section comes in. Thought it made BotStatus appear clearer.